exporter/containerimage/export.go

```diff
@@ -482,3 +482,7 @@ func addAnnotations(m map[digest.Digest]map[string]string, desc ocispecs.Descrip
 		a[k] = v
 	}
 }
+
+func defaultPlatform() string {
+	return platforms.Format(platforms.Normalize(platforms.DefaultSpec()))
```

Comment:
Please add a comment line to explain why you can’t use platforms.DefaultString()